gitlab/cli.py

```diff
@@ -299,11 +300,15 @@ def _parse_value(v: Any) -> Any:
     if isinstance(v, str) and v.startswith("@"):
         # If the user-provided value starts with @, we try to read the file
         # path provided after @ as the real value. Exit on any error.
```

Comment:
Not sure if we should keep this now as we handle specific ones:
```suggestion
        # path provided after @ as the real value.
```

Reply:
Thanks for the rework!